my_pyscf/mcscf/lasscf_rdm.py

```diff
@@ -117,48 +118,60 @@ def get_init_guess_rdm (las, mo_coeff=None, h2eff_sub=None):
 def _ddm (dm1frs0, dm1frs1):
     return np.concatenate ([(d1 - d0).ravel () for d1, d0 in zip (dm1frs0, dm1frs1)])
 
-
```

Comment:
What is the purpose of the changes to this function? It seems you replace it in your subclass anyway.